src/libs/ck-libs/io/ckio.ci

```diff
@@ -26,6 +29,53 @@ module CkIO_impl {
           entry void openFile(std::string name, CkCallback opened, Options opts);
           entry [reductiontarget] void fileOpened(FileToken file);
           entry [reductiontarget] void sessionComplete(FileToken file);
+	  // facilitates all of the read calls that will happen
+	  
+//	  entry void read(Session session, size_t bytes, size_t offset, CkCallback after_read);
```

Comment:
Remove entry methods that are commented out

Reply:
Done!